kolibri/auth/api.py

```diff
@@ -0,0 +1,109 @@
+from rest_framework import filters, permissions, viewsets
+
+from .models import (
+    Classroom, DeviceOwner, Facility, FacilityUser, LearnerGroup, Membership,
+    Role
+)
+from .serializers import (
+    ClassroomSerializer, DeviceOwnerSerializer, FacilitySerializer,
+    FacilityUserSerializer, LearnerGroupSerializer, MembershipSerializer,
+    RoleSerializer
+)
+
```

Comment:
The following two classes are the reusable permissions wrappers that we'll be importing from here into other apps for including in their viewsets. It may, therefore, be better to move them to a separate file, as we determine how we want our DRF code to be organized?